docs/usage/configuration-options.md

```diff
@@ -421,6 +421,37 @@ If enabled, all issues created by Renovate are set as confidential, even in a pu
 !!! note
     This option is applicable to GitLab only.
 
+## configMigration
+
+If enabled, Renovate will raise a pull request if config file migration is needed.
+
+As Renovate evolves and new feature are added, configuration options might evolve as well.
+For such cases, Renovate will raise config migration pull requests.
```

Comment:
```suggestion
We're adding new features to Renovate bot often.
Most times you can keep using your Renovate config and benefit from the new features right away.
But sometimes you need to change your Renovate configuration.
To help you with this, Renovate will create config migration pull requests.
```

How about something like this?

I don't like the phrases `as Renovate evolves` and `configuration options might evolve as well`. We can clarify _who's_ doing the action (us!), and also explain that most times you won't need to do anything. In the cases where config migration is needed, the bot will try to help out.